assets/lib/font-awesome/LICENSE.txt

```diff
@@ -0,0 +1,165 @@
+Fonticons, Inc. (https://fontawesome.com)
```

Comment:
Where is the preview license file?

Reply:
I don't understand, what do you mean by "preview license file"?

Reply:
Do you mean the license of font-awesome Pro? I will add it too.